notifier/chatters.rst

```diff
@@ -152,3 +152,37 @@ some interactive options called `Embed elements`_::
     $chatter->send($chatMessage);
 
 .. _`Embed elements`: https://discord.com/developers/docs/resources/webhook
+
+Adding Interactions to a Telegram Message
+-----------------------------------------
+
+With a Telegram message, you can use the
+:class:`Symfony\\Component\\Notifier\\Bridge\\Telegram\\TelegramOptions` to add
+`message options`_::
+
+    use Symfony\Component\Notifier\Bridge\Telegram\Reply\Markup\Button\KeyboardButton;
+    use Symfony\Component\Notifier\Bridge\Telegram\Reply\Markup\ReplyKeyboardMarkup;
```

Comment:
These 2 use statements seems not to relate to the used classes (`InlineKeyboardButton` & `InlineKeyboardMarkup` are used instead). Can you please check which classname is correct and fix the other one? Thanks!

Reply:
The ones used in the example are correct. I updated the `use` statements in the documentation.